### DISCUSSION TITLE
Field condition: is there a way to write it (apart the dropdown)?

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#11226] [Archive]
The field condition is great to handle whether a field should be show or hidden on detail.
But the dropdown is limited to the collection fields itself.

For instance, we would like to write
`Current_user → custom_property CONTAINS 'Auth'`

**Comment by rijkvanzanten** [comment]
So just to confirm, are you asking how to use fields on the current user, or how to write permissions without using the GUI? The title seems to be a different question than the description 😁 

To answer the title:
You can click on the field label, and use Raw Value to modify the actual JSON value that it stores.

To answer the description:
There's currently no way to use `$CURRENT_USER.x` as the _field_ in a filter for conditional fields

**Reply by Pachat** [reply]
@rijkvanzanten Sorry if I was not clear. I'll try to better explain taking the following example:

- Collection `Directus Users` has a custom field `Date of birth`.
- Collection `A` has a field `Supervision` that can be seen only if `Date of birth` is lower than `2000-01-01`

In the Data Model, the field `Supervision` in `Conditions`
- the rule only shows a drop down. I don't see any place to edit the actual JSON of the Rule?
![Supervision's Condition](https://user-images.githubusercontent.com/13746133/150830894-af7662f5-60c9-46ed-92a3-c061b4bf1f3d.png)
- and how can I show the `Supervision` field  in the Interface with the same Rule, i.e. only to the same users who can see it in Display?

_That specific supervision permission is independent from the role's permission the user already has._

**Reply by rijkvanzanten** [reply]
You can click the field label `Rule` in this case to bring up some more options on the field, including _Edit Raw Value_ 👍🏻 

I'm afraid I don't understand the second question..

**Reply by Pachat** [reply]
Ach so! Got it. Thanks!

The second question is about the `Layout Options` of the collection: 
- the user who has no rights to access the field `Supervision` should not even see it in the list of fields.

**Reply by rijkvanzanten** [reply]
> the user who has no rights to access the field Supervision should not even see it in the list of fields.

If there's no read access to a field, the user will never see it, regardless of conditional field configuration 👍🏻

**Reply by Pachat** [reply]
Yes. Thank you. That is the solution to apply to all users of a certain role.

But the problem here is that among all users of a certain role, some of them (those with a `Date of birth` before `2000-01-01`) should see the field. So, at the role level, it is necessary to keep the field.

@rijkvanzanten Thank you for your support: it helps to go further using Directus for developments.
